efb_telegram_master/cache.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+import weakref, collections, time
+
+class LocalCache():
+    notFound = None
+
+    # list dict等不支持弱引用，但其子类支持，故这里包装了下
+    class Dict(dict):
+        def __del__(self):
+            pass
+
+    def __init__(self, maxlen=20):
+        self.weak = weakref.WeakValueDictionary()
+        self.strong = collections.deque(maxlen=maxlen)
+
+    @staticmethod
+    def nowTime():
```

Comment:
`snake_case` function name.
```suggestion
    def now_time():
```